cnf-certification-test/operator/suite.go

```diff
@@ -61,3 +67,32 @@ func testOperatorInstallationPhaseSucceeded(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("Found %d CSVs whose phase is not %s.", n, v1alpha1.CSVPhaseSucceeded))
 	}
 }
+
+func testOperatorInstallationWithoutPrivileges(env *provider.TestEnvironment) {
+	badCsvs := []string{}
+	if len(env.Csvs) == 0 {
+		ginkgo.Skip("No CSVs to perform test, skipping.")
+	}
+
+	for _, csv := range env.Csvs {
+		clusterPermissions := csv.Spec.InstallStrategy.StrategySpec.ClusterPermissions
+		if len(clusterPermissions) == 0 {
+			logrus.Debugf("No clusterPermissions found in csv %s (ns %s)", csv.Name, csv.Namespace)
+			continue
+		}
+
+		for i := range clusterPermissions {
+			permission := clusterPermissions[i]
+			for ruleIndex := range permission.Rules {
+				if n := len(permission.Rules[ruleIndex].ResourceNames); n > 0 {
+					tnf.ClaimFilePrintf("CSV %s (ns %s) has cluster permissions on %d resource names.", csv.Name, csv.Namespace, n)
+					badCsvs = append(badCsvs, fmt.Sprintf("%s.%s", csv.Namespace, csv.Name))
```

Comment:
shouldn't you add break here and go to test next csv ?

Reply:
You're right. Done.